### PR TITLE
Begin citation growth architecture

### PR DESCRIPTION
**Pull request by Razzleberrytt**
## Summary

This is the first implementation batch for turning the site into a citation-oriented scientific knowledge system rather than applying isolated SEO tweaks.

- add a full citation-growth architecture specification
- add a reusable, accessible `ScientificTakeaways` MDX component for atomic citation-ready claims
- support optional evidence-confidence labels and qualifying context per takeaway
- register the component globally for article MDX
- add a copyable authoring template with claim-discipline rules
- add component tests for content, confidence labels, context, and empty-state behavior

## Why this batch comes first

The broader roadmap includes canonical definition pages, cornerstone hubs, normalized concept metadata, graph-generated relationships, structured data, and citation-readiness auditing. Those systems need a stable editorial primitive before we migrate existing articles or generate links at scale.

`ScientificTakeaways` creates that primitive while remaining opt-in and backward-compatible.

## Editorial safeguards

- extracted claims must remain accurate outside the article
- confidence describes evidence strength, not author certainty
- association must not be rewritten as causation
- vague “more research is needed” claims require a specific unresolved question
- safety instructions remain in dedicated safety blocks

## Validation

- added Vitest and Testing Library coverage
- no existing content is required to adopt the component immediately
- no schema or route behavior is changed in this first batch

### CODE REVIEW

File: components/editorial/ScientificTakeaways.tsx [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+  title = 'Scientific Takeaways',
+  summary,
+  items,
+}: ScientificTakeawaysProps) {
+  if (!items.length) return null
+
+  return (
+    <aside aria-labelledby="scientific-takeaways-title" className="my-8 rounded-3xl border border-brand-900/15 bg-brand-50/60 p-5 sm:p-6">
+      <p className="text-xs font-bold uppercase tracking-[0.16em] text-brand-700">Citation-ready summary</p>
+      <h2 id="scientific-takeaways-title" className="mt-2 text-2xl font-bold tracking-tight text-ink">
```
[the remark this answers is not in the document]
Comment (Razzleberrytt) [reply components/editorial/ScientificTakeaways.tsx]: Fixed in fcd6729 by generating a per-instance heading ID with React `useId`, and added a regression test that renders two blocks and verifies unique IDs plus matching `aria-labelledby` references.
